src/javascript/binary/pages.js

```diff
@@ -30,7 +30,11 @@ var trading_times_init = function() {
                   success: function(html){
                             trading_times.replaceWith(html);
                             trading_times = $("#trading-tabs");
-                            trading_times.tabs();
+			    if (page.language() === 'JA') {	
```

Comment:
@jy-binary we need to only display forex for trading times so all other tabs should be disabled for japan. not sure how else to do this.